static/app/views/performance/landing/index.tsx

```diff
@@ -129,19 +132,30 @@ export function PerformanceLanding(props: Props) {
                 currentFilter={spanFilter}
                 onChangeFilter={setSpanFilter}
               />
-              <SearchBar
-                searchSource="performance_landing"
-                organization={organization}
-                projectIds={eventView.project}
-                query={filterString}
-                fields={generateAggregateFields(
-                  organization,
-                  [...eventView.fields, {field: 'tps()'}],
-                  ['epm()', 'eps()']
-                )}
-                onSearch={handleSearch}
-                maxQueryLength={MAX_QUERY_LENGTH}
-              />
+              {isMetricsData ? (
+                <MetricsSearchBar
+                  searchSource="performance_landing"
```

Comment:
Would it be valuable to append `metrics` to this?

Reply:
sure...good idea